### DISCUSSION TITLE
Teensy 4.1 USB Keyboard button box for xbox?

### DISCUSSION THREAD

**Discussion by nickjdonaldson** [NicoHood/HID#352] [Q&A]
any chance of this working for xbox, trying desperately to build a small button box for flight sim on xbox series x. 

so by pressing a button, they teensy would send a simulated keyboard press. 

I'm so close, I have the xbox which see's the teensy as a keyboard but won't recognize the button presses. even though my Mac does.

**Comment by NicoHood** [comment]
This library does not support teensy
